src/SwaggerCombine.js

```diff
@@ -348,7 +348,14 @@ class SwaggerCombine {
       const conflictingOperationIds = _.intersection(operationIds, newOperationIds);
 
       if (!_.isEmpty(conflictingPaths)) {
-        throw new Error(`Name conflict in paths: ${conflictingPaths.join(', ')}`);
+        if(this.opts.continueOnConflictingPaths) {
+          const conflictingPathOps = _.intersection(_.keys(this.combinedSchema.paths[conflictingPaths]), _.keys(schema.paths[conflictingPaths]));
```

Comment:
This won't work if there is more than one conflicting path. Consider this test case:
```js
it('throws an error if continueOnConflictingPaths is true and there are multiple duplicate paths and operations', () => {
  instance.opts = { continueOnConflictingPaths: true };
  instance.schemas.push({
    paths: {
      '/test/path/first': {
        get: {
          summary: 'GET /test/path/first duplicate',
        },
      },
      '/test/path/second': {
        get: {
          summary: 'GET /test/path/first duplicate',
        }
      },
    },
  });

  expect(instance.combineSchemas.bind(instance)).to.throw(/Name conflict in paths: \/test\/path\/first at operation: get/);
});
```

In the current implementation no error is thrown for this test case. You'll have to find a way to check all conflicting paths for conflicting operations.